File: Api/HipotecaSimple/Data/Entities/Usuarios.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace HipotecaSimple.Data.Entities
{
    public class Usuarios
    {
        [Key]
        public int UsuID { get; set; }
        public short UsuStatus { get; set; }
        public string UsuNombre { get; set; }
        public string UsuSesion { get; set; }
        public string UsuPass { get; set; }
        public string UsuPermisos { get; set; }
        public DateTime UsuFechaCreacion { get; set; }

        [NotMapped]
        public string Token { get; set; }
    }
}

File: Web/src/app/Entities/Usuarios.ts
export class Usuarios {
  usuID: number;
  usuStatus: number;
  usuNombre: string;
  usuSesion: string;
  usuPass: string;
  empId: number;
  usuFechaCreacion: Date;
  token: string | null;

  constructor() {
      this.usuID = 0;
      this.usuStatus = 0;
      this.usuNombre = '';
      this.usuSesion = '';
      this.usuPass = '';
      this.empId = 0;
      this.usuFechaCreacion = new Date();
      this.token = null;
  }
}

export class LoginViewModel {
  user: string;
  password: string;

  constructor() {
      this.user = '';
      this.password = '';
  }
}